resources/load-statically-from-npm.js

```diff
@@ -4,9 +4,9 @@
 
 const fs = require('fs');
 
+const ts = require('typescript');
+
```

Comment:
Ideally, we shouldn't require typescript since our typescript can conflict with typescript that is calling transformation.
It looks like context contain factory for creating nodes:
https://github.com/microsoft/TypeScript/blob/eb3645f16b66ed05110ea957c77773cea739bae8/src/compiler/transformers/es2019.ts#L4
Not sure but maybe it also allows accessing predicate functions.